components/event-bus/api/push/eventing.kyma-project.io/v1alpha1/types.go

```diff
@@ -70,4 +70,8 @@ const (
 	Ready SubscriptionConditionType = "is-ready"
 	// EventsActivated label for the Kyma subscription condition type.
 	EventsActivated SubscriptionConditionType = "events-activated"
+	// ChannelReady label for Knative channel readiness in Kyma Subscription
+	ChannelReady SubscriptionConditionType = "channel-ready"
```

Comment:
For consistency, does it make sense to also prefix this with `kn-`?

Reply:
Fixed.